sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/RecognizeIDDocumentOptions.java

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.models;
+
+import com.azure.core.annotation.Fluent;
+
+import java.time.Duration;
+import java.util.List;
+
+import static com.azure.ai.formrecognizer.implementation.Utility.DEFAULT_POLL_INTERVAL;
+
+/**
+ * Options that may be passed when using recognize ID Document APIs on Form Recognizer client.
+ */
+@Fluent
+public final class RecognizeIDDocumentOptions {
+    private FormContentType contentType;
+    private boolean includeFieldElements;
+    private List<String> pages;
+    private Duration pollInterval = DEFAULT_POLL_INTERVAL;
+
+    /**
+     * Get the type of the form. Supported Media types including .pdf, .jpg, .png or .tiff type file stream.
+     *
+     * @return the {@code contentType} value.
+     */
+    public FormContentType getContentType() {
+        return contentType;
+    }
+
+    /**
+     * Get the boolean which specifies if to include form element references in the result.
+     *
+     * @return the {@code includeFieldElements} value.
+     */
+    public boolean isFieldElementsIncluded() {
+        return includeFieldElements;
+    }
+
+    /**
+     * Get the duration between each poll for the operation status. If none is specified, a default of
+     * 5 seconds is used.
+     *
+     * @return the {@code pollInterval} value.
+     */
+    public Duration getPollInterval() {
+        return pollInterval;
+    }
+
+    /**
+     * Set the type of the form. Supported Media types including .pdf, .jpg, .png or .tiff type file stream.
+     *
+     * @param contentType the provided form content type.
+     *
+     * @return the updated {@code RecognizeBusinessCardOptions} value.
```

Comment:
edit: same edit to the docstrings below

```suggestion
     * @return the updated {@code RecognizeIDDocumentOptions} value.
```